internal/annotation/env/env.go

```diff
@@ -0,0 +1,93 @@
+package env
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/spoditor/spoditor/internal/annotation"
+	corev1 "k8s.io/api/core/v1"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	MountEnv = "mount-env"
+)
+
+var log = logf.Log.WithName("mount_env")
+
+type mountEnv struct {
+	qualifier string
+	cfg       *mountConfigValue
+}
+
+type mountConfigValue struct {
+	EnvFrom    []corev1.EnvFromSource `json:"envFrom"`
```

Comment:
According to the [Pod spec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables) relevant to how environment variable(s) can be specified for the container(s), I think we only need the `Containers` field, because both `EnvFrom` and `Env` are child fields of `container`, for example:

```yaml
annotations:
  spoditor.io/env_1-: |
    {
      "containers": [
        {
          "name": "nginx",
          "env": [
            {
              "name": "FIXED",
              "value": "I'm present on all pods except the first one"
            },
            {
              "name": "REF_KEY_FROM_SECRET_WITH_ORDINAL_SUFFIX",
              "valueFrom":
                {
                  "secretKeyRef:
                    {
                      "name": "mysecret",
                      "key": "nameOfKey"
                    }
                }
            }
          ]
          "envFrom": [
            {
              "configMapRef":
                {
                  "name": "myconfigmap"
                }
            }
          ]
        }
      ]
    }
```
In such way, we can generically support all three different ways of specifying environmental variables.